.stickler.yml

```diff
@@ -0,0 +1,4 @@
+linters:
+  rubocop:
+display_cop_names: true
+fixer: true
```

Comment:
Hmmm, I am not sure this the best thing to do. You should review the linting errors and try to fix those you missed.

See below for recommended .stickler.yml file.
This would help you with some unreasonable linting errors.
```yml
linters:
  rubocop:
    display_cop_names: true
files:
  ignore:
    - "bin/*"
    - "db/*"
    - "config/*"
    - "Guardfile"
    - "Rakefile"
```